convert2rhel/main.py

```diff
@@ -100,6 +100,9 @@ def main():
         loggerinst.task("Final: rpm files modified by the conversion")
         systeminfo.system_info.modified_rpm_files_diff()
 
+        # Remove temporary folder with backup - last step of conversion
+        utils.remove_tmp_dir()
```

Comment:
```suggestion
        loggerinst.task("Final: Remove temporary folder %s" % TMP_DIR)
        utils.remove_tmp_dir()
```
Without this, the info log message printed by `remove_tmp_dir` would look like they relate to the previous task ("Final: rpm files modified by the conversion").